File: Codes/PageId.php
<?php 

	class PageId {

		private $_fileId;
		private $_idx; 

		public function __construct($fileId, $idx) {
			$this->setFileId($fileId);
			$this->setIdx($idx);
		}

		public function setFileId ($fileId) { $this->_fileId=$fileId; }
		public function setIdx ($idx) { $this->_idx=$idx; }

		public function getFildeId () { return($this->_fileId); }
		public function getIdx () { return($this->_idx); }
		
	}
 ?>

File: ProjetBDDA/src/managers/PageId.java
package managers;

public class PageId {

	public int fileId;
	public int idx;

	public PageId(int fileId, int idx) {
		this.setFileId(fileId);
		this.setIdx(idx);
	}

	private void setFileId(int fileId) {
		this.fileId = fileId;
	}

	private void setIdx(int idx) {
		this.idx = idx;
	}

	public int getFileId() {
		return (this.fileId);
	}

	public int getIdx() {
		return (this.idx);
	}

}
